scripts/remote-tmux-live-fuzz.sh

```
@@ -249,19 +294,105 @@ check_screen_oracle() {
+  ' 2>/dev/null)
+  if [ "$(printf '%s\n' $on_screen_windows | grep -c .)" -gt 1 ]; then
+    # Same re-read discipline as every other judge: a probe can land inside
+    # the one-frame handoff of a tab switch, and only a state that persists
+    # through a fresh census is a defect.
+    # Judging the stale snapshot again would just repeat the first read, so
+    # a failed refresh defers the verdict to the next iteration entirely.
+    if refresh_pane_surfaces; then
+      on_screen_windows=$(printf '%s' "$PANE_SURFACES_JSON" | jq -r '
+        [.panes[]? | select(.on_screen == true) | .window_id] | unique | join(" ")
+      ' 2>/dev/null)
+      if [ "$(printf '%s\n' $on_screen_windows | grep -c .)" -gt 1 ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)